src/Common/Commands.Common/MetricHelper.cs

```diff
@@ -207,9 +207,11 @@ public void FlushMetric()
         /// <returns></returns>
         public static string GenerateSha256HashString(string originInput)
         {
-            SHA256 sha256 = SHA256.Create();
-            var bytes = sha256.ComputeHash(Encoding.UTF8.GetBytes(originInput));
-            return BitConverter.ToString(bytes);
+            using (var sha256 = new SHA256CryptoServiceProvider())
```

Comment:
@shahabhijeet There are a set of providers that are FIPS compliant, and a set that are not.  If you use one of the FIPS-compliant providers, then there is no exception when FIPS mode is turned on: https://blogs.msdn.microsoft.com/icumove/2009/01/31/working-with-fips-in-net-c/